build-image/first-boot.bash

```diff
@@ -134,21 +137,17 @@ sleep 10  # Related to: https://github.com/openhab/openhabian/issues/441#issueco
 echo "OK"
 
 echo -n "$(timestamp) [openHABian] Updating repositories and upgrading installed packages... "
-/usr/bin/apt-get --yes --fix-broken install &>/dev/null
-apt-get --yes upgrade &>/dev/null
-if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
+apt-get --yes --fix-broken install &>/dev/null
+if apt-get --yes upgrade &>/dev/null; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
 
 if hash python 2>/dev/null; then bash /boot/webif.bash reinsure_running; fi
 
 echo -n "$(timestamp) [openHABian] Installing git package... "
-/usr/bin/apt-get -y install git &>/dev/null
-if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
+if apt-get -y install git &>/dev/null; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
 
 echo -n "$(timestamp) [openHABian] Cloning myself... "
-[ -d /opt/openhabian/ ] && rm -rf /opt/openhabian/ # check if we have remnants of a previous installation attempt.
-git clone -b master https://github.com/openhab/openhabian.git /opt/openhabian &>/dev/null
-
-if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
```

Comment:
@ThomDietrich I agree that this can feel a bit more foggy yes, but I think it worth the tradeoff. With this method we can ensure all commands are checked by using `&&` when running commands. I think we should rewrite stuff as the example below. This is both programmatically better and quite readable. I would though like to do that in an succeeding PR. wdyt?  

```
if \
  command1 && \
  command 2 && \
  command 3 ; \
then echo "OK"; else echo "FAILED"; exit 1; fi
```

Reply:
Agree but please do it in a separate PR.
This one is 144 comments already which is really a top scorer so let's get this thing finally merged.